packages/terra-clinical-onset-picker/docs/README.md

```diff
@@ -0,0 +1,28 @@
+# Terra Clinical Onset Picker
+
+The terra-clinical-onset-picker component provides users a way to enter or select an approximate date for use in onset scenarios.
+
+This component manages its own state.
```

Comment:
Don't think we need to include this in the description.

Reply:
Remove